lib/linguist/languages.yml

```diff
@@ -5240,6 +5240,19 @@ Pike:
   tm_scope: source.pike
   ace_mode: text
   language_id: 287
+Pkl:
+  type: programming
+  color: "#6b9543"
+  extensions:
+  - ".pkl"
+  - ".pcf"
```

Comment:
We need samples and supporting searches for all extensions and filenames added to a PR. If usage isn't sufficient, support can be added later and an override can be used in the interim.

Also keep in mind that the same grammar and thus syntax highlighting will be applied to all extensions and filenames associated with a language.

Reply:
> We need samples and supporting searches for all extensions and filenames added to a PR. If usage isn't sufficient, support can be added later and an override can be used in the interim.

Oh, I guess I missed that and thought they could be lumped together as they are just different extensions/names for the same language.
What do you mean by `an override can be used in the interim.`?

> Also keep in mind that the same grammar and thus syntax highlighting will be applied to all extensions and filenames associated with a language.

That is fine. Both `.pcf` and `PklProject` are just Pkl files. `pcf` is a subset of Pkl.

Reply:
> What do you mean by `an override can be used in the interim.`?

https://github.com/github-linguist/linguist/blob/master/docs/overrides.md